packages/experiments/src/components/Pagination/Pagination.base.tsx

```diff
@@ -0,0 +1,233 @@
+            disabled={!canFirst}
+            aria-label={firstPageAriaLabel}
+            styles={{
+              icon: this._classNames.previousNextPage,
+              rootDisabled: this._classNames.previousNextPageDisabled
+            }}
+          />
+          <IconButton
+            iconProps={this.props.previousPageIconProps}
+            onClick={this.handlePreviousPage}
+            disabled={!canPrevious}
+            aria-label={previousPageAriaLabel}
+            styles={{
+              icon: this._classNames.previousNextPage,
+              rootDisabled: this._classNames.previousNextPageDisabled
+            }}
+          />
+          {this._pageList()}
+          <IconButton
+            iconProps={this.props.nextPageIconProps}
+            onClick={this.handleNextPage}
+            disabled={!canNext}
+            aria-label={nextPageAriaLabel}
+            styles={{
+              icon: this._classNames.previousNextPage,
+              rootDisabled: this._classNames.previousNextPageDisabled
+            }}
+          />
+          <TooltipHost content={`${pageAriaLabel} ${pageCount + 1}`} directionalHint={DirectionalHint.bottomCenter}>
+            <IconButton
+              iconProps={this.props.lastPageIconProps}
+              onClick={this.handleLastPage}
+              disabled={!canLast}
+              aria-label={lastPageAriaLabel}
+              styles={{
+                icon: this._classNames.previousNextPage,
+                rootDisabled: this._classNames.previousNextPageDisabled
+              }}
+            />
+          </TooltipHost>
+        </div>
+        {visibleItemLabel && (
+          <div className={this._classNames.visibleItemLabel} aria-label={visibleItemLabel}>
+            {visibleItemLabel}
+          </div>
+        )}
+      </div>
+    );
+  }
+
+  private handleFirstPage = () => {
+    this.handleSelectedPage(0);
+  };
+
+  private handleLastPage = () => {
+    this.handleSelectedPage(this.props.pageCount - 1);
+  };
+
+  private onComboBoxChanged = (option: IComboBoxOption, index: number, value: string) => {
+    if (option !== undefined) {
+      this.handleSelectedPage(index);
+    }
+  };
+
+  private handleSelectedPage = (selected: number) => {
+    const { selectedPageIndex, onPageChange } = this.props;
+    if (selected === selectedPageIndex) {
+      return; // same page, no action
+    }
+    if (onPageChange) {
+      onPageChange(selected);
+    }
+  };
+
+  private handlePreviousPage = () => {
+    this.handleSelectedPage(this.props.selectedPageIndex! - 1);
+  };
+
+  private handleNextPage = () => {
+    this.handleSelectedPage(this.props.selectedPageIndex! + 1);
+  };
+
+  private _pageElement(index: number): JSX.Element {
+    const { pageAriaLabel, pageCount, selectedPageIndex, selectedAriaLabel } = this.props;
+    const isSelected = index === selectedPageIndex;
+    let ariaLabel = pageAriaLabel && `${pageAriaLabel} ${index + 1} of ${pageCount}`;
```

Comment:
again, careful that you're embedding english here

Reply:
please see my other comment

---
In reply to: [233575229](https://github.com/OfficeDev/office-ui-fabric-react/pull/7100#discussion_r233575229) [](ancestors = 233575229)